src/sentry/static/sentry/app/views/settings/incidentRules/types.tsx

```diff
@@ -118,6 +118,14 @@ export type MetricActionTemplate = {
    * Integration id for this `type`, should be passed to backend as `integrationId` when creating an action
    */
   integrationId: number;
+  /**
+   * The input type on the front end, e.g. text for Slack, select for PagerDuty
+   */
+  inputType?: string;
+  /**
+   * If the inputType is select, these are the options to select from
+   */
+  options: Array<Object> | null;
```

Comment:
@scefali is this better? I'm super new to typescript so I wasn't sure if I need to also define the types of the objects' values.

Reply:
@ceorourke it's better than before, but it would be even better to specify what are the types in each object.